apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -68,6 +69,23 @@ const passwordFormatter = (loginType, {rowData}) => {
   );
 };
 
+// The "add row" should always be pinned to the top when sorting.
+// This function takes into account having multiple "add rows"
+const sortRows = (data, columnIndexList, orderList) => {
+  let addRows = [];
+  let studentRows = [];
+  for (let i = 0; i<data.length; i++) {
+    if (data[i].isAddRow) {
+      addRows.push(data[i]);
+    } else {
+      studentRows.push(data[i]);
+    }
+  }
+  addRows = orderBy(addRows, columnIndexList, orderList);
+  studentRows = orderBy(studentRows, columnIndexList, orderList);
+  return addRows.concat(studentRows);
```

Comment:
This seems like it could be optimized to be done in a single call to `orderBy`, but some playing around with the command line at https://lodash.com/docs/4.17.4#orderBy did not immediately reveal the solution. Maybe worth investigating at some point in the future if sorting is slow.

Reply:
Good idea I'll keep this in mind when I add the "add multiple students" functionality